projects/v2/src/app/app-config.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable }  from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { ReplaySubject } from 'rxjs';
+import { SheetService } from './services/sheet.service';
+import { SheetDetails } from './models/sheet.model';
+ 
+@Injectable()
+export class ConfigService {
+
+    config = new ReplaySubject(2);
```

Comment:
config => config$

Reply:
config$ = this.http.get<Record<string, unknown>>('assets/configuration.json').pipe(shareReplay(1));